src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -26,23 +31,89 @@ public void parse_invalidArgs_throwsParseException() {
     }
 
     @Test
-    public void parse_validArgs_returnsFindCommand() {
+    public void parse_validNameArgs_returnsFindCommand() {
         // no leading and trailing whitespaces
+        NameContainsKeywordsPredicate nameContainsKeywordsPredicate =
+                new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob"));
```

Comment:
Any reason why you didn't import the valid person's name and/or substring the name to avoid regressions?

Reply:
The same question goes for their details like address, contact, email, etc.

Reply:
I feel its better for readability, to see the actual inputs, actual and expected results.